README.md

```diff
@@ -41,6 +41,46 @@ The Image Tool provides three functions within the main script:
 
 Use the [Quick Start](site/quickstart.md) guide to create a Linux based WebLogic Docker image.
 
+## Building From Source
+
+The Image Tool installer is available for download on the [Releases](releases) page.  If you wish
+to build the installer from source instead of downloading it, please follow these instructions:
+- Download and install JDK 8u261+
+- Download and install Maven 3.6.3+
+- Clone this repository to your local environment using one of the options under `Code` near the top of this page.
+- From inside the top-level directory of the cloned project, `weblogic-image-tool`, using Maven, execute one or 
+more of these phases:
+    - `validate` - validate the project is correct and all necessary information is available.
+    - `compile`  - compile the source code.
+    - `test`     - test the compiled source code using the JUnit5 framework.
+    - `package`  - create the installer ZIP, `imagetool.zip`.
+    - `verify`   - run integration tests using the JUnit5 framework.
+    - `clean`    - restore the source by removing any items created by `package` or another phase of the build.
+    
+**Note:** Maven executes build phases sequentially, `validate`, `compile`, `test`, `package`, `verify`, such that 
+running `verify` will run all of these phases from `validate` through `package` before executing `verify`.
+
+Since the `package` phase comes before the `verify` phase, it is not necessary to run the integration tests to create 
```

Comment:
Since -> Because 
(since – don't use to mean for the reason of; use because. Use since only for matters of time, for example, since the last time the program was run.)